backend/unit/unit.go

```diff
@@ -0,0 +1,41 @@
+package unit
+
+import (
+	"errors"
+	"strconv"
+	"time"
+)
+
+const (
+	oneDay  = time.Hour * 24
+	oneWeak = oneDay * 7
+)
+
+// ParseDuration reads a human-readable duration and returns duration in seconds
+// TODO: extend to ms or add other units, such as storage (B, KB, MB, GB, TB, PB) and temperature ( C, F).
```

Comment:
Here is our TODO format: TODO format: // TODO(issue#631): support responsive layouts.